libs/net/dal/Migrations/1.0.137/Up/PostUp/00-ReportTemplate.sql

```diff
@@ -0,0 +1,290 @@
+DO $$
```

Comment:
All links need to be correctly formatted so that they work in both the browser and email.